en/docs/install-and-setup/setup/sso/okta-as-an-external-idp-using-oidc.md

```diff
@@ -1,65 +1,73 @@
-In the following document we will explain how to connect OKTA as a third party Identity provier to WSO2 API-Manager. Before we start first make sure you have all the pre-requisites mentioned below.
+# Using OKTA as an External IDP with OIDC
+
+This section explains how to connect OKTA as a third-party Identity provier to WSO2 API Manager.
 
 ### Pre-requisites
 
+Before you begin, make sure you do the following.
+
 1. Create an account in [https://developer.okta.com/](https://developer.okta.com/)
-2. Download the latest API-Manager distribution from [https://wso2.com/api-management/](https://wso2.com/api-management/)
-3. Unzip the distribution and open the `deployment.toml` file located in `<APIM_HOME>/repository/conf/` and add the following configuration
+2. Download the latest API Manager distribution from [https://wso2.com/api-management/](https://wso2.com/api-management/)
+3. Unzip the distribution and open the `deployment.toml` file located in `<APIM_HOME>/repository/conf/` folder. Add the following configuration
     ```
     [tenant_mgt]
     enable_email_domain= true
     ```
-    This is needed since OKTA uses the email as the username by default,  therefore to  use the email as the username in WSO2 API-Manager we have to enable it since it not enabled by default. Once enabled both email as username and normal usernames can be used.
+    You need to enable this because OKTA uses the email as a username by default. To  use the email as the username in WSO2 API Manager you have to enable it, since it not enabled by default. Once enabled, you can use your email or a normal username as your username.
```

Comment:
"OKTA uses the email as a username by default" -> OKTA uses the email as the username by default